src/polyfills.ts

```diff
@@ -37,8 +37,8 @@
 /** IE10 and IE11 requires the following to support `@angular/animation`. */
 // import 'web-animations-js';  // Run `npm install --save web-animations-js`.
 /** Evergreen browsers require these. **/
-import "core-js/es6/reflect";
-import "core-js/es7/reflect";
+import "core-js/es/reflect";
```

Comment:
This doesn't seem to be related to your feature of this PR. On my local machine this change breaks the build please reset it to the version that's in the master branch.